packages/api-server/db_migrations/006_zone_info.sql

```diff
@@ -0,0 +1,31 @@
+ALTER TABLE zones ADD is_deep_road BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
I should have asked this on earlier commits, but does the migration tool you are using create transactions for you, or is there a reason for not putting each new migration file in a transaction? Just a safeguard that will likely never get hit but is nice to have

Reply:
I'm not sure what you mean. You mean by terminating the line with a `;`? 

AFAIK, a transaction is an open connection to the db. This should be creating a transaction in that sense, but I could be misunderstanding.